Credere.postman_collection.json

```diff
@@ -0,0 +1,354 @@
+{
+  "info": {
+    "_postman_id": "01b37b1e-3706-43d4-a18b-6a34a5a880f0",
+    "name": "Credere",
+    "schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+    "_exporter_id": "15689195"
+  },
+  "item": [
+    {
+      "name": "http://127.0.0.1:8000/users/register",
+      "request": {
+        "method": "POST",
+        "header": [
+          {
+            "key": "Authorization",
+            "value": "AWS4-HMAC-SHA256 Credential=AKIAQJHNKG5JU6EYV2GB secret: BBL4gcPjsv+92Rv1HRQp5qVPxabA5azgilyU83YUXz/e/20230516/us-east-1/cognito-idp/aws4_request, SignedHeaders=content-type;host;x-amz-date;x-amz-target, Signature=2d462d62639727959651311cd798de73dd330d16519950a30e05ebca947bb885",
```

Comment:
Should this be here? @Lombardoh @nahu

Reply:
It should not be here. I've deleted this postman collection [here](https://github.com/open-contracting/credere-backend/pull/51/commits/281dab643b5ccb7c1f80b5a6a9ee3de074b427bc)